src/Processors/Transforms/FilterSortedStreamByRange.cpp

```diff
@@ -0,0 +1,51 @@
+#include <Processors/Transforms/FilterSortedStreamByRange.h>
+
+#include <Columns/IColumn.h>
+#include <Interpreters/ExpressionActions.h>
+
+namespace DB
+{
+
+FilterSortedStreamByRange::FilterSortedStreamByRange(
```

Comment:
It's a bit tricky to catch. Is there any code change together with moving?

Reply:
Nope. It's separated to debug this issue so that changing this file won't need to recompile PartSplitter.